forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -543,6 +552,37 @@ export default function CreateNewFilterDrawer({
     setShowUndefinedVariable(showUndefinedVariable);
   };
 
+
+  const hideDeleteConfirmation = () => {
+    setShowDeleteModal(false);
```

Comment:
Please ensure that clicking the cancel button on the modal triggers the display of the drawer.

Reply:
okay,will check and update